src/mimir_coordinator.py

```diff
@@ -69,6 +70,10 @@ def is_coherent(self) -> bool:
         roles: Iterable[MimirRole] = self._cluster_provider.gather_roles().keys()
         return set(roles).issuperset(MINIMAL_DEPLOYMENT)
 
+    def is_scaled(self) -> bool:
```

Comment:
since we're checking whether the workers are scaled, I'd do instead:
`number_of_workers(self) -> int` so you can write more explicit/transparent code to use it

Reply:
changed the name to has_multiple_workers but retained the logic, tbh I'm not sure if returning the actual number of workers would add anymore benefit cause mimir would need s3 if it's more than more worker anyways